Libraries/Hotcakes.Commerce.Dnn/Hotcakes.Commerce.Dnn.csproj

```diff
@@ -1,5 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\..\packages\EntityFramework.6.4.4\build\EntityFramework.props" Condition="Exists('..\..\packages\EntityFramework.6.4.4\build\EntityFramework.props')" />
```

Comment:
Is this supposed to be here?  More important... Why is it here?  Any data access should be coming from the `Hotcakes.Commerce` project and the other libraries/projects should be calling into that.  Otherwise, there would have been a reference here that was replaced.  This one was added, so it wasn't necessary before and probably isn't necessary now.  

We need to be careful to keep the solution clean.  Do you have a good technical reason to add this circular dependency?  If not, it should be removed and re-tested.

Reply:
I have performed the EntityFramework update using the Nuget-Package-Manager tool, it seems that when performing the update automatically, unnecessary code segments have been added. I have made the updates again only changing what is necessary and everything has worked correctly.

Reply:
Add a suggestion, <Ctrl+g>

New! Suggest specific code changes that the pull request author or assignees can immediately commit. You will be attributed in the commit.

Reply:
I have performed the EntityFramework update using the Nuget-Package-Manager tool, it seems that when performing the update automatically, unnecessary code segments have been added. I have made the updates again only changing what is necessary and everything has worked correctly.

Reply:
This pull request needs some changes before being approved.